test/Sema/access-level-import-conformances.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file --leading-lines %s %t
+
+/// Build the libraries.
+// RUN: %target-swift-frontend -emit-module %t/ConformanceBaseTypes.swift -o %t
+// RUN: %target-swift-frontend -emit-module %t/ConformanceDefinition.swift -o %t -I %t
+
+/// Check diagnostics.
+// RUN: %target-swift-frontend -typecheck -verify %t/Client.swift -I %t \
+// RUN:   -enable-experimental-feature AccessLevelOnImport
+
+//--- ConformanceBaseTypes.swift
+public protocol Proto {}
+public struct ConformingType {
+    public init () {}
+}
+
+//--- ConformanceDefinition.swift
+import ConformanceBaseTypes
+extension ConformingType : Proto  {}
+
+//--- Client.swift
+public import ConformanceBaseTypes
+internal import ConformanceDefinition
+
+public func useInAPI(a: any Proto = ConformingType()) { // expected-error {{cannot use conformance of 'ConformingType' to 'Proto' here; 'ConformanceDefinition' was not imported publicly}}
```

Comment:
Should there be a corresponding `note:` on the import decl?

Reply:
Very good point. I've added the note.